sdk/src/main/java/io/radar/sdk/RadarBeaconManager.kt

```diff
@@ -602,8 +602,11 @@ internal class RadarBeaconManager(
     }
 
     private fun isBluetoothSupported(context: Context): Boolean {
-        if (!this::adapter.isInitialized) {
-            adapter = BluetoothAdapter.getDefaultAdapter()
+        if(!this::adapter.isInitialized) {
+            val defaultAdapter = BluetoothAdapter.getDefaultAdapter()
+            if (defaultAdapter != null) {
```

Comment:
We may want to bring this logic to the other call sites where `adapter` gets initialized, but as long as we're always guarding those with calls to `isBluetoothSupported()` we should be ok.